bb_proxy/src/lib.rs

```diff
@@ -112,14 +112,7 @@ impl Proxy {
               ver.id()
             } as i32,
           },
-          players: JsonPlayers {
-            max:    69,
-            online: 420,
-            sample: vec![JsonPlayer {
-              name: "macmv".into(),
-              id:   "a0ebbc8d-e0b0-4c23-a965-efba61ff0ae8".into(),
-            }],
-          },
+          players: JsonPlayers { max: 1024, online: 0, sample: vec![] },
```

Comment:
can you make the max of 1024 configurable? it shouldn't be enforced, but 1024 is just a random constant, and I'd like to avoid that

Reply:
right, but having a config for that is not worth the hassle for now imo, so I put set it to 0. bamboo will most likely be used behind a proxy so it shouldn't matter, we should implement that along with the online count. 0/0 is better than 420/69 lol